CHANGELOG.md

```diff
@@ -1,26 +1,268 @@
+| Updated styleguide colors & typography                           | **Update**        | [#2236][] |
+| Removed an unwanted link                                         | **Update**        | [#2202][] |
+| Corrected mispelling in 'Introduction to React'                  | **Bugfix**        | [#2487][] |
+| Fixed Modal/Portal reference doc generation                      | **Bugfix**        | [#2450][] |
+| Fixed content in Cloud Deploy topic                              | **Bugfix**        | [#2398][] |
+
+### Misc
+
+| Description                                                               | Change type | PR                  |
+| ------------------------------------------------------------------------- | ----------- | ------------------- |
+| Bumped `workbox` dependency to 5.1.2                                      | **Update**  | [#2480][]           |
+| Bumped `websocket-extensions` dependency from 0.1.3 to 0.1.4              | **Update**  | [#2465][]           |
+| Bumped `apollo-server` dependency from 2.6.9 to 2.14.2                    | **Update**  | [#2464][]           |
+| Added Lars Roettig to the list of Community Maintainers                   | **Update**  | [#2439][] [#2293][] |
+| Updated pull request template to use valid markdown syntax for checkboxes | **Update**  | [#2434][]           |
+| Updated docker image                                                      | **Update**  | [#2406][]           |
+| Updated pull request template                                             | **Update**  | [#2389][]           |
+| Renamed docker file to get syntax highlighting                            | **Update**  | [#2374][]           |
+| Bumped jquery from 3.4.1 to 3.5.0 in pwa-devdocs project directory        | **Update**  | [#2370][]           |
+| Updated the node version in venia-concept to allow Node >=10.x            | **Update**  | [#2315][]           |
+| Bumped acorn from 5.7.3 to 5.7.4                                          | **Update**  | [#2312][]           |
+| Updated `minimist` dependency to >= 1.2.2                                 | **Update**  | [#2267][]           |
+| Fixed `https-proxy-agent` dependency issue                                | **Bugfix**  | [#2356][]           |
+
+## Known issues
+
+- A new Mini Cart is in development which removes the checkout flow in favor of the standalone checkout page.
+  An issue exists in the checkout section of the old Mini Cart where the **Pay with Card** header does not appear on the Credit Card payment form.
+  This issue is low priority since the old Mini Cart will be replaced with the new one in future releases.
 
 ## Upgrading from a previous version
 
-The method for updating to 6.0.1 from a previous version depends on how PWA Studio is incorporated into your project.
+The method for updating to 7.0.0 from a previous version depends on how PWA Studio is incorporated into your project.
```

Comment:
We need a better answer than "This method can be a chore, and we hope that some of the features in 5.0.0+ will help these users migrate to a package management approach." What is our current advice? Surely it's evolved since 5.0.0.

Reply:
We publish packages that developers can list as dependencies, and upgrading those is just a matter of updating the `package.json` file in their project. This is the easiest and preferred way of using PWA Studio.

However, other developers have manually copied over code from this project into their own or used a fork of this repository to build their storefronts. Since that code has been modified by the developer, they have to manually apply any updates and/or fix merge conflicts.